pkg/boot/grub/grub.go

```diff
@@ -126,6 +133,29 @@ func grubScanBLSEntries(mountPool *mount.Pool, variables map[string]string) ([]b
 	return images, nil
 }
 
+// Find and return the value of the key from a grubenv file
+func findkeywordGrubEnv(file string, fsRoot string, key string) (string, error) {
+	FileGrubenv, err := filepath.Glob(filepath.Join(fsRoot, file))
+	if FileGrubenv == nil || err != nil {
+		return "", err
+	}
+	relNamesFile, err := os.Open(FileGrubenv[0])
+	if err != nil {
+		return "", fmt.Errorf("[grubenv] Could not Open %s: %v", FileGrubenv[0], err)
```

Comment:
note that it is not necessary to print the name of the file, os.Open will include that in the error return.

e.g.
func main() {
	_, err := os.Open("nofile")
	fmt.Printf("%v\n", err)
}
rminnich@pop-os:/tmp$ go run a.go
open nofile: no such file or directory

You can see that when I print the error, it includes the file name. 
You can simplify this return
return "", fmt.Errorf("[grubenv]:%w", err)

Reply:
Done.